docpipeline/generate.py

```diff
@@ -225,14 +221,13 @@ def build_and_format(blob, is_bucket, devsite_template):
     return tmp_path, metadata, site_path
 
 
-def get_base_url(language, name):
-    # The baseUrl must start with a scheme and domain. With no scheme, docfx
-    # assumes it's a file:// link.
-    base_url = f"https://cloud.google.com/{language}/docs/reference/" + f"{name}/"
-    # Help packages should not include the version in the URL.
-    if name != "help":
-        base_url += "latest/"
-    return base_url
+def get_path(metadata):
+    path = f"/{metadata.language}/docs/reference/{metadata.name}"
+    if metadata.stem != "":
+        path = metadata.stem
```

Comment:
Do we have plans to include any language specific guides that's not under `/docs/reference`? For a hypothetical example, `https://coud.google.com/python/guides`.

Reply:
It's possible. The stem would be `/python/guides` in that case. But, no particular plans at the moment.

Reply:
In case you're thinking about non-guide content, we are planning on reference docs for App Engine that aren't under `/lang/docs/reference`.

Reply:
Got it. Sounds good, I think we can live with that for now as there aren't going to be that many we should keep in mind of.